hail/python/hailtop/hailctl/dataproc/resources/init_notebook.py

```diff
@@ -39,14 +39,15 @@ def mkdir_if_not_exists(path):
     pip_pkgs = [
         'setuptools',
         'mkl<2020',
-        'ipywidgets<8',
-        'nbconvert<6',
-        'notebook<6',
-        'qtconsole<5',
         'lxml<5',
-        'google-cloud==0.32.0',
+        'google-cloud==0.32.*',
         'jgscm<0.2',
-        'jupyter', 'tornado<6',  # https://github.com/hail-is/hail/issues/5505
```

Comment:
Is the version of tornado that we install fixed?

Reply:
This can be removed. I updated the issue with a comment to this effect. Jupyter fixed their bugs.

Reply:
Erm, to answer your question, we don't need tornado, the version requirement here was to fix a missing version requirement in jupyter.